### ISSUE TITLE
Settings Dialog: aViewer becomes non-reponsive/hangs on Windows 7

### ISSUE THREAD

**Issue by jgauvreau** [ThePacielloGroup/aviewer#27]
aViewer becomes non-responsive if I open the Settings window and attempt to change any settings (see screenshot below):
![image](https://cloud.githubusercontent.com/assets/12006767/24554386/56cefcfc-15fb-11e7-83a7-7ef3afd07924.png)

The Windows dialog that appears has the following information about the error:
> Description:
>   A problem caused this program to stop interacting with Windows.
> 
> Problem signature:
>   Problem Event Name:	AppHangB1
>   Application Name:	aViewer.exe
>   Application Version:	2.0.1.69
>   Application Timestamp:	58de0cda
>   Hang Signature:	bcc2
>   Hang Type:	0
>   OS Version:	6.1.7601.2.1.0.256.4
>   Locale ID:	1033
>   Additional Hang Signature 1:	bcc21320a354a490c52571a6d3952e86
>   Additional Hang Signature 2:	237a
>   Additional Hang Signature 3:	237acb176da410d9ef3397fa1ef5313a
>   Additional Hang Signature 4:	bcc2
>   Additional Hang Signature 5:	bcc21320a354a490c52571a6d3952e86
>   Additional Hang Signature 6:	237a
>   Additional Hang Signature 7:	237acb176da410d9ef3397fa1ef5313a
> 
> Read our privacy statement online:
>   http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409
> 
> If the online privacy statement is not available, please read our privacy statement offline:
>   C:\WINDOWS\system32\en-US\erofflps.txt
>

**Comment by patrickhlauke** [comment]
Cannot repro on Windows 10. Just to check, could you unregister and reregister the DLL from that dialog, then close and reopen aViewer?

**Comment by jgauvreau** [comment]
@patrickhlauke Unfortunately, for me on Windows 7, it seems to happen no matter what I do once the Settings window is opened. 

I just tried to unregister the DLL. After I clicked the Unregister button with my mouse and then tried to click anywhere else in the window and the Settings window became non-responsive. 
![image](https://cloud.githubusercontent.com/assets/12006767/24557314/43249b80-1605-11e7-993d-860ec8df0c87.png)

I forced aViewer to close, reopened and went to the Settings window, clicked the Register button with my mouse and then tried to click anywhere else in the window and the Settings window became non-responsive again.
![image](https://cloud.githubusercontent.com/assets/12006767/24557402/94383ef0-1605-11e7-8c6e-48a5b50f73e7.png)

I forced aViewer to close again, reopened and went to the Settings window and then immediately clicked the Cancel button. The Settings window did not close as expected and when I tried to click anywhere in the parent aViewer window then it again became non-responsive.
![image](https://cloud.githubusercontent.com/assets/12006767/24557503/e9d2da0a-1605-11e7-8e98-550c3bee6ee0.png)

Same issue occurred if I tried to click the OK button immediately after opening the Settings window.
Same issue occurred if I open aViewer and just try tabbing with the keyboard into the interactive elements in the Settings window.
